phpunit/class-wp-theme-json-resolver-test.php

```diff
@@ -358,4 +358,164 @@ public function test_get_user_data_from_wp_global_styles_does_not_use_uncached_q
 		$user_cpt = WP_Theme_JSON_Resolver_Gutenberg::get_user_data_from_wp_global_styles( $theme );
 		$this->assertEmpty( $user_cpt, 'User CPT is expected to be empty.' );
 	}
+
```

Comment:
This is part of https://github.com/WordPress/gutenberg/pull/45969 To be removed when that PR lands.